optimum/bettertransformer/models/base.py

```diff
@@ -61,6 +63,13 @@ def __init__(self, config):
                 self.num_layers = getattr(config, attr)
                 break
 
+        if old_layer is not None:
+            # Last step, store the old module skeleton by copying the old module and putting
+            # it on the `meta` device.
+            self.old_layer = deepcopy(old_layer).to("meta")
+        else:
+            self.old_layer = old_layer
+
```

Comment:
The meta allows to avoid x2 in memory, right?

Reply:
yes